lib/pod/command/dependencies.rb

```diff
@@ -122,12 +124,26 @@ def graphviz_data
         @graphviz ||= begin
           require 'graphviz'
           graph = GraphViz::new(output_file_basename, :type => :digraph)
-          root = graph.add_node(output_file_basename)
 
-          unless @podspec
-            podfile_dependencies.each do |pod|
-              pod_node = graph.add_node(pod)
-              graph.add_edge(root, pod_node)
+          unless @use_podfile_targets
```

Comment:
please don't use `unless` with `else`, prefer `if` instead

Reply:
Ok, I will make the change.

Reply:
Fixed via 73a6b0b